nautobot_ssot/jobs/examples.py

```diff
@@ -252,6 +316,60 @@ def delete(self):
         return super().delete()
 
 
+class PrefixLocalModel(PrefixModel):
+    """Implementation of Prefix create/update/delete methods for updating local Nautobot data."""
+
+    @classmethod
+    def create(cls, diffsync, ids, attrs):
+        """Create a new Prefix record in local Nautobot.
+
+        Args:
+            diffsync (NautobotLocal): DiffSync adapter owning this Prefix
+            ids (dict): Initial values for this model's _identifiers
+            attrs (dict): Initial values for this model's _attributes
+        """
+        prefix = Prefix(prefix=ids["prefix"], description=attrs["description"])
+        tenant_obj, _ = Tenant.objects.get_or_create(
+            slug=ids["tenant_slug"],
+            defaults={"name": ids["tenant_slug"]},
+        )
+        prefix.tenant = tenant_obj
```

Comment:
Don't we need to handle the case where tenant_slug is None?

Reply:
this is true, I will have to convert to None tenant object when the tenant_slug is ""